specification/streamanalytics/resource-manager/Microsoft.StreamAnalytics/preview/2017-04-01-preview/subscriptions.json

```diff
@@ -134,6 +234,68 @@
         }
       },
       "x-ms-azure-resource": true
+    },
+    "OperationResult": {
+      "description": "The result of the requests to fetch the operation results.",
+      "allOf": [
+        {
+          "$ref": "../../common/v1/definitions.json#/definitions/Error"
+        }
+      ],
+      "properties": {
```

Comment:
Is there supposed to also be an "error" object here for the cases where there is an error query testing unrelated to us so we say query testing succeeded (200) but it resulted in the following error? For example, in test connection, if we can't connect to their input/output due to a bad password or something, we don't return 400 or 500, we would return 200 (test connection succeeded) but the result of the test connection status is failed with the error being the password provided was incorrect

This might be a question for Ken